xbmc/video/VideoThumbLoader.cpp

```diff
@@ -400,7 +400,12 @@ bool CVideoThumbLoader::FillLibraryArt(CFileItem &item)
     const CURL itemUrl{item.GetPath()};
     if (itemUrl.HasOption("videoversionid"))
     {
-      if (m_videoDatabase->GetArtForAsset(tag.m_iFileId, tag.m_iDbId, tag.m_type, artwork))
+      if (m_videoDatabase->GetArtForAsset(tag.m_iFileId, tag.m_iDbId, tag.m_type,
+                                          item.HasProperty("noartfallbacktoowner") &&
+                                                  item.GetProperty("noartfallbacktoowner") == true
+                                              ? false
+                                              : true,
```

Comment:
Not 100% sure, but imo the whole term could be shortened to `!item.GetProperty("noartfallbacktoowner").asBoolen(false)`.

Reply:
yes. that's convenient!